specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -29260,6 +29669,116 @@
         "modelAsString": true
       }
     },
+    "WorkflowEnvelope": {
+      "description": "Workflow properties definition.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource id."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the resource name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets the resource type."
+        },
+        "kind": {
+          "type": "string",
+          "description": "The resource kind."
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location."
+        },
+        "properties": {
+          "description": "Additional workflow properties.",
+          "type": "object",
+          "properties":{
+            "files": {
+              "description": "Gets or sets the files.",
+              "type": "object",
+              "additionalProperties": {
+                "type": "object"
```

Comment:
Can you please define the schema correctly ? free form objects are not allowed

Reply:
What is the protocol for free form objects? In this case we write a file containing connection data that's defined on a connector by connector basis. There are ~800 total connectors, each with its own connection parameters: https://learn.microsoft.com/en-us/connectors/connector-reference/connector-reference-logicapps-connectors

Reply:
Bump, any clarification?